apps/modernization-ui/src/apps/page-builder/page/management/edit/question/QuestionHeader.tsx

```diff
@@ -27,6 +30,7 @@ export const QuestionHeader = ({
     onDeleteQuestion,
     visible = true
 }: Props) => {
+    const editModalRef = useRef<ModalRef>(null);
```

Comment:
Issue(blocking): Modal should be created in PageContent, so each question on the page doesn't have its on modal created.